app/mm-redux/actions/remote_cluster.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {Client4} from '@mm-redux/client';
+import {RemoteClusterTypes} from '@mm-redux/action_types';
+
+import {bindClientFunc} from './helpers';
+export function getRemoteClusterInfo(remote_id: string) {
+    return bindClientFunc({
+        clientFunc: Client4.getRemoteClusterInfo,
+        onRequest: RemoteClusterTypes.REMOTE_CLUSTER_INFO_REQUEST,
+        onSuccess: [RemoteClusterTypes.REMOTE_CLUSTER_INFO_SUCCESS, RemoteClusterTypes.REMOTE_CLUSTER_INFO_RECEIVED],
```

Comment:
We are not reacting to it but thought could be useful in logging and handling `_FAILURE` (UI/UX) .. as of now, we are silently failing/not showing the info in case request fails.

Reply:
@anurag6713 think of this, for each request this will update the store three times, causing the mapStateToProps from every mounted component to re-execute and potentially some selectors to re-compute.. is it really needed? I'd say definitely not and we should remove thr Request trackers... if you want to react to a failure then I invite you to change the action and then return a data and error object and react based on that.

Do not just use something that seems like a pattern just because, everything should have a reason behind it.

Definitely change this

Reply:
Agreed! Did it by following the pattern and will remove it.